runtests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#rm -rf env
+virtualenv env
+. env/bin/activate
+pip install -e .
+pip install nose mock coverage
+nosetests --with-coverage --nocapture calico.openstack.test.test_plugin:TestPlugin
```

Comment:
You almost certainly want to remove 'nocapture' here. This is also a really confusing file to add to the repository because it only tests one thing, rather than testing the full set. It would be better to add a Makefile with a `test` target that covers what we do in Jenkins.